### PR TITLE
Time Series algorithms enumerated

### PR DESCRIPTION
**Pull request by kuanhc96**
-created algorithms_ts.py under src/decanter/core/enums to list available algorithms for user to specify in BuildSpec
-added "algorithms" parameter to TrainTSInput constructor and BuildSpec

### CODE REVIEW

File: examples/auto_time_series_example.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from decanter import core
 from decanter.core.core_api import TrainTSInput, PredictTSInput, SetupInput
 from decanter.core.enums.algorithms import Algo
+#from decanter.core.enums.algorithms_ts import AlgoTS
```
Comment (CliffLin) [inline examples/auto_time_series_example.py:9]: remove this line directly

File: src/decanter/core/core_api/train_input.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 from decanter.core.core_api import CoreBody
 from decanter.core.enums.algorithms import Algo
+#from decanter.core.enums.algorithms_ts import AlgoTS
```
Comment (CliffLin) [inline src/decanter/core/core_api/train_input.py:11]: ditto

File: src/decanter/core/core_api/train_input.py [hunk 3]
```diff
@@ -122,6 +122,7 @@ def __init__(
             time_groups=None, max_window_for_feature_derivation=None):
 
         evaluator = check_is_enum(Evaluator, evaluator)
+        #algorithms = [check_is_enum(AlgoTS, algo) for algo in algorithms]
```
Comment (CliffLin) [inline src/decanter/core/core_api/train_input.py:125]: ditto

File: src/decanter/core/enums/__init__.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 """Init jobs package"""
```
Comment (CliffLin) [inline src/decanter/core/enums/__init__.py:1]: please help to update the header description

File: examples/auto_time_series_example.py [hunk 5]
```diff
@@ -21,7 +21,7 @@ def main():
     # Create connection to Decanter server, and set up basic settings.
     # Logger message:
     #   "[Context] connect healty :)" if success.
-    client = core.CoreClient(username='gp', password='gp-admin', host='http://localhost:3000')
+    client = core.CoreClient(username='gp', password='gp-admin', host='http://192.168.2.7:3000')
```
Comment (jwolf903) [inline examples/auto_time_series_example.py:24]: Change this back to localhost

File: src/decanter/core/extra/utils.py [hunk 6]
```diff
@@ -14,6 +14,7 @@ def check_response(response, key=None):
     """
     code = response.status_code
     if not 200 <= code < 300:
+        print('response: ' + str(response))
```
Comment (CliffLin) [inline src/decanter/core/extra/utils.py:17]: use logging instead of print

Comment (kuanhc96) [reply src/decanter/core/extra/utils.py]: This was used for debugging. I just removed it!

File: src/decanter/core/jobs/data_upload.py [hunk 7]
```diff
@@ -25,6 +25,7 @@ class DataUpload(Job):
     """DataUpload manage to get the result from data upload.
 
     Handle the execution of upload task in order to upload data to
+
```
Comment (CliffLin) [inline src/decanter/core/jobs/data_upload.py:28]: what is it

Comment (kuanhc96) [reply src/decanter/core/jobs/data_upload.py:28]: not sure what you mean. Should I remove the white line?

Comment (jwolf903) [reply src/decanter/core/jobs/data_upload.py:28]: yes :)

File: examples/jupyter_example.ipynb [hunk 8]
```diff
@@ -615,7 +615,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.8.5"
```
[the remark this answers is not in the document]
Comment (kuanhc96) [reply examples/jupyter_example.ipynb]: removed this file and replaced with original -- should resolve the issue

File: src/decanter/core/enums/algorithms_ts.py [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+    - Prophet: Prophet forecasting model for univariate time series datasets
+    - Theta: Theta forecasting model
+    - LSTM: Google's LSTM.
+    - XGBoost: eXtreme Gradient Boosting.
+    """
+    Prophet = 'Prophet'
+    Theta = 'Theta'
+    LSTM = 'LSTM'
+    XGBoost = 'XGBoost'
```
Comment (jwolf903) [inline src/decanter/core/enums/algorithms_ts.py:21]: This isn't used anymore right?

**Review by jwolf903 — CHANGES_REQUESTED** [review]

**Review by jwolf903 — DISMISSED** [review]

**Review by jwolf903 — APPROVED** [review]

**Review by CliffLin — APPROVED** [review]

**Pull request merged**